liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/RunServerMojo.java

```diff
@@ -65,7 +65,14 @@ private void doRunServer() throws MojoExecutionException {
         boolean hasDownstreamProjects = false;
         ProjectDependencyGraph graph = session.getProjectDependencyGraph();
         if (graph != null) {
-            checkMultiModuleConflicts(graph);
+        	
+        	// In a multi-module build, the run server goal will only be run on one project (the farthest downstream) and compile will
+        	// be run on any relative upstream projects. If this current project in the Maven Reactor is not one of those projects, skip it.  
+        	List<MavenProject> devModeProjects = getRelevantMultiModuleProjects(graph);
+        	if (!devModeProjects.contains(project)) {
+        		getLog().info("\nSkipping run goal.\n");
```

Comment:
Should this info log message include something to identify which project is being skipped...like the project id? Also, nit pick but why is the collection called devModeProjects? We are not in dev mode here.

Reply:
Fixed the variable naming - didn't catch after copy and paste :)